backend/.env.development

```diff
@@ -67,11 +67,27 @@ LANGCHAIN_API_KEY=your_langsmith_api_key_here
 # To create a LangSmith project, visit LangSmith: https://www.langchain.com/langsmith
 LANGCHAIN_PROJECT=your_langsmith_project_name_here
 
-
 # FILE_UPLOAD: Whether to enable file upload to storage
-# Set to true if file upload is required.
+# Available options: false, s3, minio
 # If set to false, AWS_S3_BUCKET_NAME is not required.
```

Comment:
It seems that this comment should be changed because the available options are changed.